libraries/AP_Logger/AP_Logger_File.cpp

```diff
@@ -1058,7 +1063,24 @@ void AP_Logger_File::_io_timer(void)
         ::fsync(_write_fd);
         last_io_operation = "";
 #endif
+
+#if CONFIG_HAL_BOARD == HAL_BOARD_CHIBIOS
+        // ChibiOS does not update mtime on writes, so if we opened
```

Comment:
I'm surprised by this (it should do it on fsync, not writes, but anyway). Is that a ChibiOS issue or a FatFS problem? I tried searching for it but couldn't find anything.